### PR TITLE
refactor web application docs

### PR DESCRIPTION
**Pull request by HarshCasper**

### CODE REVIEW

**Review by whummer — APPROVED** [review]
LGTM! 👍 Minor suggestions - maybe @lukqw can also take a brief look  👀

File: content/en/user-guide/web-application/accounts/index.md [hunk 1]
```diff
@@ -1,22 +1,29 @@
 ---
 title: "Accounts"
-tags: [] 
+tags: ["LocalStack Pro", "Accounts"]
 weight: 2
 description: >
-  LocalStack accounts are the entry point to our web application, the Pro CLI, and more!
+  LocalStack Accounts allows you to manage your resources & configurations, and serves as an entry point to our Web Application, the Pro CLI, and more!
```
Comment (lukqw) [inline content/en/user-guide/web-application/accounts/index.md:6]: `allow`
`serve`

File: content/en/user-guide/web-application/teams/index.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+---
+title: "Teams"
+tags: [] 
+weight: 2
+description: >
+  LocalStack Organizations/Teams offers a flexible way of managing access roles and permissions for your team members.
```
Comment (lukqw) [inline content/en/user-guide/web-application/teams/index.md:6]: `offer`

Comment (lukqw) [reply content/en/user-guide/web-application/teams/index.md:6]: Maybe add something about API Key Management here as well?

File: content/en/user-guide/web-application/teams/index.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+---
+
+Teams on LocalStack allow you to collaborate with members of your organization. You can invite other users to your team and assign them different roles and permissions. It includes advanced role settings such as purchasing subscriptions, managing payment methods, and more by the team administrator. You can purchase a plan with multiple seats for your team, which allows you to distribute the API keys among your team members.
```
Comment (lukqw) [inline content/en/user-guide/web-application/teams/index.md:9]: `It includes advanced role settings such as purchasing subscriptions, managing payment methods, and more, which can be adjusted by the team administrator.`

**Review by lukqw — COMMENTED** [review]
Awesome work, thanks! 💪 
Added a couple of nits, other than that it's good to go from my side.

(I see this has already been merged)

**Pull request merged**